torch/_dynamo/variables/torch.py

```diff
@@ -122,13 +123,16 @@ def remap_as_fn___rxor__(*args):
 def remap_as_fn___rand__(*args):
     return torch._C.TensorBase.__rand__(*args)
 
+def remap_as_fn_is_contiguous():
+    return torch.Tensor.is_contiguous()
 
 tensor_dunder_fns_remap = {
     torch._C.TensorBase.__radd__: remap_as_fn___radd__,
     torch._C.TensorBase.__rmul__: remap_as_fn___rmul__,
     torch._C.TensorBase.__ror__: remap_as_fn___ror__,
     torch._C.TensorBase.__rxor__: remap_as_fn___rxor__,
     torch._C.TensorBase.__rand__: remap_as_fn___rand__,
+    torch._C.TensorBase.is_contiguous: remap_as_fn_is_contiguous,
```

Comment:
Pretty sure this is not the right place to put this.

That bing said, whatever is the fix, it should support (or break properly) when `memory_format` is provided.